<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhasorWidget</class>
 <widget class="QWidget" name="PhasorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Phasor Map</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <widget class="ColormappedImageWidget" name="image_widget" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="channel_combo">
       <property name="currentText">
        <string>Channel 0</string>
       </property>
       <item>
        <property name="text">
         <string>Channel 0</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="display_combo">
       <item>
        <property name="text">
         <string>Selected Image</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>All Images</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColormappedImageWidget</class>
   <extends>QWidget</extends>
   <header>ColormappedImageWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
